app/controllers/providers/allocations_controller.rb

```diff
@@ -49,6 +49,17 @@ def show
       end
     end
 
+    def new_request
```

Comment:
Can we use the 'new' action which is already defined? This would be more RESTFUL also ...

Reply:
Actually thinking about it some more, it seems like 'requesting PE again' should be edit / update and requesting PE for the first time should be new / create ... Might need some more heads around this.

It seems like 'requesting PE for an organisation' might be a candidate for a 'wizard' (multi-step form)

Reply:
More discussion ensues - https://ukgovernmentdfe.slack.com/archives/CGVNJ2Q2F/p1587991640283900